plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/TestProject.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.appengine.localserver.server;
+package com.google.cloud.tools.eclipse.test.util.project;
```

Comment:
Not a fan of this class name.  I think it should indicate that it's a faceted project.  `FacetedProjectCreator`?

Do the todos in the class comment still apply?